ngi_visualizations/fpkm_scatter/fpkm_scatter.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/python
+"""
+fpkm_scatter.py
+
+Takes FPKM counts from two conditions and makes a scatter plot.
+Also calculates a r-squared correlation score.
+
+Either takes two Cufflinks FPKM summary files or a summary file
+with multiple samples.
+"""
+
+from __future__ import print_function
+
+import argparse
```

Comment:
And to be honest, IMHO we should not care about compatibility with Python <= 2.6... unless of course the package **is** written in some version of Python <= 2.6, which is not the case for ngi_visualizations